.solhint.json

```diff
@@ -4,7 +4,7 @@
     "indent": ["error", 4],
     "quotes": ["error", "double"],
     "max-line-length": ["error", 129],
-    "function-max-lines": ["error", 60],
+    "function-max-lines": ["error", 62],
```

Comment:
Are we going to keep adjusting this in every PR? Maybe we should either disable it or chose a reasonably large value?

Reply:
Or refactor :rocket:

Reply:
Jokes. Yes I would be happy to carefully evaluate the existing functionality of the submit solution and potentially break it into something smaller and more reasonable. Maybe even a modifier for all the initial/elementary constraints

Reply:
@nlordell any chance we can get rid of this change?